insights/parsers/sys_kernel.py

```diff
@@ -0,0 +1,38 @@
+"""
+SysKernel - file ``/proc/sys/kernel`` or ``/sys/kernel/``
```

Comment:
Generally, the first item in the title should be a `ParserClass`, when it contains multiple Parsers, better use as `Descriptive title for the parsers`

See https://github.com/RedHatInsights/insights-core/wiki#checklist-for-reviewers